libraries/standard/mqtt/include/mqtt.h

```diff
@@ -22,7 +22,7 @@
 #ifndef MQTT_H
 #define MQTT_H
 
-#include "config.h"
+#include "mqtt_config.h"
```

Comment:
Why do we need to rename `config.h`?

Reply:
Instead of using an aggregate config file (i.e. configurations for all libraries in a single config file), config files have been broken into library-specific config files (like `shadow_config.h`, `http_config.h`, etc) to allow for library-specific log configuration.